src/EcomDestinationWriter.cs

```diff
@@ -4055,25 +4106,23 @@ private bool HasData(string tableName)
     private string GetValue(ColumnMapping columnMapping, Dictionary<string, object> row)
```

Comment:
I am afraid having this method changed to concatenate the other mapping columns values, this method is used in 17 places so I am pretty sure it can break something somewhere. Can you make an overload version of this method or a new method and use it in your implementation where it is needed?

This method is used for searching existing entities like products by number so having it returned the concatenate values will break that functionality

Reply:
ok, i've changed it so a new function is made and i call this where i need it.